guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: guide
+title: Liquidity
+description: A primer on lightning network liquidity
+nav_order: 3
+parent: How it works
+permalink: /guide/how-it-works/liquidity/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/liquidity/liquidity-preview.jpg
+---
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-title.jpg"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-title@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-title-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-title-mobile@2x.jpg"
+   alt-text = "Photo of an ocean with the word 'Liquidity'"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+<!--
+
+Editor's notes
+
+Figma file: https://www.figma.com/file/ZwRT4mZZ4UNGtsfLz22m8M/Liquidity?node-id=0%3A1
+
+-->
+
+# Liquidity
+
+Liquidity is a core concept to understand when working with the lightning network, although tricky. Ideally, we are able to design products that are easy enough to use so that users do not need to concern themselves with liquidity. However, product designers need to understand liquidity in order to build effectively on lightning.
+
+## What is liquidity?
+
+Put simply, liquidity on the lightning network is the ability to send or receive bitcoin. In the context of an individual user, liquidity is a measurement of their wallet's ability to send and receive bitcoin over lightning.
+
+### Inbound vs Outbound Liquidity
+
+*Inbound liquidity* is the amount of bitcoin that the user is able to receive over a lightning channel. *Outbound liquidity* is the amount of bitcoin that the user is able to send over a lightning channel.
+
+Receiving bitcoin decreases the user's inbound liquidity and increases their outbound liquidity. On the other hand, sending bitcoin decreases their outbound liquidity and increases their inbound liquidity.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound-mobile@2x.png"
+   alt-text = "Graphic depicting inbound liquidity flowing towards a lightning node and outbound liquidity flowing away from a lightning node"
```

Comment:
As the concept of a node hasn't been introduced by this stage of the page, I would change the title under the graphic from 'Lightning node' to 'Lighting user.'

Reply:
I don't follow. We use the term "lightning node" 34 times in the Guide across 15 separate pages (not including what's baked into images). On all but 2 of the pages, we never "introduce" the concept of a lightning node, we simply use the term lightning node.

If this is problematic, perhaps we could link to one of the pages that goes into more detail about a lightning node, such as [Software](https://bitcoin.design/guide/getting-started/software/#nodes), [Hardware](https://bitcoin.design/guide/getting-started/hardware/), or [Tech Primer](https://bitcoin.design/guide/getting-started/technology-primer/#the-lightning-payment-network), or even add a glossary entry for the term "lightning node".

Regardless, if this is an actually an issue, it's an issue on 13 other pages and not just this one.

Reply:
I'm about to open a PR adding lighting content to the `Glossary` -> `Node` page, could add a link to that from this page, as well as others. 

I do think the term node just coming up can confuse people not familiar with the concept, so we should address that in other pages (most likely just linking to the node page when its merged).